card_test.go

```diff
@@ -141,6 +141,33 @@ func (card *Card) fakeGetRulings(rulingNumber int) string {
 	return strings.Join(ret, "\n")
 }
 
+func fakeSearchResults(input CardSearchResult) ([]Card, error) {

```

Comment:
Actually we should just separate out the fetch part from the parse and format and return logic, so that we don't have to mock any of our logic (and thus keep two identical functions up to date)